libphidgets/package.xml

```diff
@@ -1,25 +1,21 @@
-<package>
+<?xml version="1.0"?>
+<package format="2">
   <name>libphidgets</name>
   <version>0.6.1</version>
   <description>This package wraps the libphidgets to use it as a ros dependency</description>
-  <!-- <maintainer>Florian Weisshardt</maintainer> -->
 
   <license>LGPL</license>
 
   <url type="website">http://ros.org/wiki/libphidgets</url>
-  <!-- <url type="bugtracker"></url> -->
   <maintainer email="benjamin.maidel@ipa.fraunhofer.de">Benjamin Maidel</maintainer>
   <author>Alexander Bubeck</author>
 
   <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>libusb-dev</build_depend>
   <build_depend>mk</build_depend>
-  <build_depend>rosbuild</build_depend>
   <build_depend>rospack</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>libusb-dev</build_depend>
-  <run_depend>libusb</run_depend>
 
-  <export>
-    <cpp lflags="-L${prefix}/lib -lphidget21" cflags="-I${prefix}/include"/>
-  </export>
+  <exec_depend>libusb</exec_depend>
```

Comment:
You exported libusb-dev as `DEPENDS` in catkin_package(), thus it needs to be an exec_depend, too...
Simply combine build_depend libusb-dev and exec_depend libusb to depend libusb-dev!